inits.json

```diff
@@ -32,6 +32,7 @@
 
             "Camera Type (CCD or DSLR)": "CCD",
             "Pixel Binning": "1x1",
+            "Exposure Time": "60",
```

Comment:
Fixed this in my previous pull request. Marlee took it out due to us now taking exposure times from FITS headers.